### DISCUSSION TITLE
Stato sensore temperatura sconosciuto/non disponibile

### DISCUSSION THREAD

**Discussion by Assistnew** [indomus/forum#2842] [GENERALE - Discussioni generiche]
Buon pomeriggio,
sono alle prime armi con HA e ho un fastidioso problema che non riesco a risolvere.
Un sensore di temperatura (sonda ds18b20 su shelly add-on), da qualche tempo, risulta spesso in stato sconosciuto o non disponibile.
Il sensore (sensor.temperatura_interna_temperature) è legato all'entità termostato e causa, passando in stato unknown, un malfunzionamento dello stesso.
Come potrei risolvere il problema?
Questa la configurazione del termostato:

#Termostato  
climate:
  - platform: generic_thermostat
    name: Riscaldamento
    heater: switch.shellyplus1_02387
    target_sensor: sensor.temperatura_interna_temperature
    min_temp: 10
    max_temp: 25
    ac_mode: false
    target_temp: 18
    cold_tolerance: 0.3
    hot_tolerance: 0.3
    min_cycle_duration:
      seconds: 5
    keep_alive:
      minutes: 1
    initial_hvac_mode: "off"
    away_temp: 16
    precision: 0.1

Grazie!

**Comment by 50494554524F** [comment]
puoi derivare un sensore di tipo trigger che escluda unavailable/unknown, in questo modo tutto rimane congelato... userai questo nuovo sensore con il termostato; lo stato dei sensori trigger si conserva con il riavvio, quindi tutto rimarrà consistente

chiaramente trova un modo di rendere visibile l'indisponibilità del sensore originario così da poter verificare subito se ci sono problemi
